shared/fixes/bash/auditd_data_disk_full_action.sh

```diff
@@ -1,16 +1,10 @@
 # platform = multi_platform_rhel
+
+# Include source function library.
 . /usr/share/scap-security-guide/remediation_functions
+
 populate var_auditd_disk_full_action
 
-#
-# If disk_full_action present in /etc/audit/auditd.conf, change value
-# to var_auditd_disk_full_action, else
-# add "disk_full_action = $var_auditd_disk_full_action" to /etc/audit/auditd.conf
-#
+cpe_number='27370-6'
```

Comment:
why this change?

Reply:
I think that it increases readability at no cost.

Reply:
In this shorthand script, sure. But in HTML output, I don't think that it does.

Reply:
we have a macro for CCEs, they must not be hardcoded in the scripts

Reply:
and it's cce, not cpe, so this makes it super confusing

Reply:
If I recall correctly, making this change would break that macro, but I haven't tried it to confirm.

Reply:
OK, I have rebased and put the `@CCENUM@` macro back there.